packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -343,6 +344,25 @@ export class Account {
   async listCommunityBalances (): Promise<CommunityBalanceData> {
     return await this.client.call('listcommunitybalances', [], 'bignumber')
   }
+
+  /**
+   * Returns information about burn history
+   *
+   * @param {BurnHistoryOptions} [options]
+   * @param {number} [options.maxBlockHeight]  The block height to iterate from.
+   * @param {number} [options.depth] Maximum depth, from the genesis block is the default
+   * @param {string} [options.token] Filter by token
+   * @param {DfTxType} [options.txtype] Filter by transaction type. See DfTxType.
+   * @param {number} [options.limit=100] Maximum number of records to return, 100 by default
```

Comment:
can you add test for `maxBlockHeight`, `depth` and `limit`??

Reply:
Sure, let me add that